tests.py

```diff
@@ -1,7 +1,56 @@
 import reddit_news_scrap
 from requests_html import HTMLSession
+from collections import namedtuple
 
 def test_ensure_200():
     session = HTMLSession()
     response = session.get("https://microsoft.com")
     assert reddit_news_scrap.is_200(response)
+
+
+def test_articles_empty():
+    articles = []
+    try:
+        reddit_news_scrap.articles_is_empty(articles)
+        assert False
+    except SystemExit as e:
+        assert True
+
+
+def test_articles_not_empty():
+    articles = ["test"]
+    try:
+        reddit_news_scrap.articles_is_empty(articles)
+        assert True
+    except SystemExit as e:
+        assert False
+
+class MockArticle:
+    
+    def __init__(self,absolute_links,text):
+        self.text = text
+        self.absolute_links = absolute_links
+
+    def __getitem__(self, position):
+        return self.absolute_links[position]
+
+def test_article_attributes():
+    #Link = namedtuple("Link", "text")
```

Comment:
it appears as though namedtuples aren't being used anymore.  We should problem remove them.